examples/icon-position.js

```diff
@@ -119,3 +116,8 @@ if (pivot) {
   const axis = camera.right;
   rotateAroundAxis(camera, -Math.PI / 4, axis, transform, options);
 }
+
+//##REMOVE## Keep this tag, split code here for code sandbox
+
+import {initCodeSandbox} from './_code-sandbox.js';
+initCodeSandbox('rawjs/icon-position.js');
```

Comment:
Could we pass data/image-static.png and data/icon.png here to avoid base64-ing them?